app/gui/docs/product/visualizations.md

```diff
@@ -201,24 +200,11 @@ In particular:
   as visualizations. The superclass defines a default constructor and a set of
   utilities:
 
-  - #### Method `setPreprocessorCode(code)`
-    Set an Enso code which will be evaluated on the server-side before sending
-    data to visualization. If not called, a default unspecified code is used
+  - #### Method `setPreprocessor(module,method,arguments)`
```

Comment:
It should be clarified that `arguments` are vararg rather than a singular argument.